Adapt memberName for clock terminal variable within high-cut (#220)
modelica/fmi-ls-bus: docs/3____physical_signal_abstraction.adoc

 * There must be no `<TerminalStreamMemberVariable>` element.
 * There must be one <<high-cut-pdu-terminal>> element per PDU of this frame.
 * There must be one `<TerminalMemberVariable>` for the Clock this frame is connected to.

>    The `memberName` of this variable shall be equal to the value `TransmissionClock`.

Reviewer: ```suggestion
   The `memberName` of this variable must be `TransmissionClock`.
```
